app/views/admin/orders/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1><%= @store.name.titleize  %> 's Orders </h1>
+<% @orders.count_by_status.each do |key, value| %>
+  <h3>  <%= key.titleize %> : <%= value %></h3>
+<% end %>
+<div class="container">
+  <center>
+    <table class="table table-hover">
+      <thead>
+        <tr>
+          <th>Order #</th>
+          <th>Order Placed</th>
+          <th>Order Status</th>
+          <th>Total</th>
+          <th>Update</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @orders.each do |order| %>
+          <tr id="order-<%= order.id %>" class="order">
+            <th scope="row"> <%= link_to order.id,  admin_store_order_path(@store.url, order) %></th>
+            <td><%= order.date %></td>
+            <td><%= order.status.capitalize %></td>
+            <!-- <td>$<%= order.total_price %></td> -->
+            <td>
+              <% sum = 0 %>
+              <% order.items.each do |item| %>
+                <% sum += (item.price * OrderItem.find_by(item: item, order: order).quantity) %>
```

Comment:
we probably need a presenter for this...this is a lot of view logic

Reply:
I agree- this is a chore ticket already on the icebox.